src/pages/jeongilLee/Login/Login.js

```diff
@@ -1,14 +1,96 @@
-import React, { Component } from 'react';
+import React from 'react';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: '',
+      pw: '',
+      buttonChange: 'logInButton',
+    };
+  }
+
+  handleIdInput = e => {
+    this.setState({
+      id: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pw: e.target.value,
+    });
+  };
```

Comment:
```suggestion
  handleInputs = e => {
    this.setState({
      [e.target.name]: e.target.value,
    });
  };
```
위와 같이 하나의 함수를 사용하는 방법도 있습니다.
저렇게 하려면 input태그에 name값을 지정해줘야 해요!
console.log()를 통해서 값이 어떻게 오는지 확인해보면서 리팩토링 진행해주세요 :)